contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -380,8 +380,8 @@ select CAST(c.oid as int) as object_id
   , case when a.attnotnull then CAST(0 as sys.bit) else CAST(1 as sys.bit) end as is_nullable
   , CAST(0 as sys.bit) as is_ansi_padded
   , CAST(0 as sys.bit) as is_rowguidcol
-  , CAST(0 as sys.bit) as is_identity
-  , CAST(0 as sys.bit) as is_computed
+  , case when a.attidentity = 'a' then CAST(1 as sys.bit) else CAST(0 as sys.bit) end as is_identity
```

Comment:
Those views are handled already.

Reply:
Instead of comparing attidentity with 'a', please update the condition to check if the value is not null because attidentity can take the values 'a' and 'g':
`CAST(case when a.attidentity <> ''::"char" then 1 else 0 end AS sys.bit) `